docs/scheduler/README.md

```diff
@@ -3,38 +3,29 @@
 ## Overview
```

Comment:
I remember when I tried to install scheduler, it asked me to create an app engine application first which is weird.

Reply:
do you remember that step? Do you know if there is a gcloud command for that? 
I think we will have to dogfood all of these guides. If things don't work, then we will have to add the missing steps...

Reply:
In my case, I just went to gcp console and created an app engine application manually. There is no gcloud command. I have not investigated why Scheduler needs an app engine application.

Reply:
OK, I found the doc and updated it... Thanks for that catch @capri-xiyue